gdb2cm.go

```diff
@@ -53,33 +56,58 @@ func readDashboardJson(file string) *models.Dashboard {
 	return dbo
 }
 
-func main() {
-	kingpin.Parse()
-	if !strings.HasSuffix(*dashboardFile, ".json") {
-		panic(fmt.Sprintf("%s is not a file...exiting", *dashboardFile))
+func processDir(dbDirGlob, mDir string) {
+	dirs, err := filepath.Glob(dbDirGlob)
```

Comment:
this is just going to return a list of files right? Not specifically directories?  Is the glob expected to only match directories, or do you need post filter the matches to only directories?